FIJI/BatchStitchPooledCellPainting_StitchAndCrop_Headless.py

```diff
@@ -130,7 +164,9 @@
                         IJ.run("Grid/Collection stitching", standard_grid_instructions[0] + filename + standard_grid_instructions[1])
                         im=IJ.getImage()
                         #We're going to overwrite this file later, but it gives is a chance for an early checkpoint
-                        IJ.saveAs(im,'tiff',os.path.join(out_subdir,fileoutname))
+                        #This doesn't seem to play nicely with the compression option on, it doesn't get overwritten later and bad things happen
```

Comment:
Do I understand correctly that we save this file so that we can manually look at stitching progress before it is finished in case we want to abort a bad run? And that doesn't play well with compression?

Reply:
Yup; you would need to be running locally though, or log into the machine while it's running, which in practice we basically never do now anyway.